cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -561,6 +561,10 @@ func (mh *MessageHandle) saveSuccessPoint(msg *beehiveModel.Message, info *model
 					ObjectName:       resourceName,
 				},
 			}
+			if objectSync.Spec.ObjectKind == "" {
+				klog.Errorf("Failed to init objectSync: %s, ObjectKind is empty, msg content: %v", objectSyncName, msg.GetContent())
+				return
```

Comment:
Last question maybe, with `util.SetMetaType(obj.(runtime.Object))` in kubeClientGet, I think the kind will almost never empty, right?

Reply:
yep, i think so. Dynamiccontroller informers alse already called SetMetaType in eventhandler. Cloudhub send other type msgs to edge by calling ListMessageWriteLoop, which no need to create objectsync, such as device twin, applicationResponse, user data.

Reply:
Sounds good.